modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts

```diff
@@ -5,8 +5,8 @@
  * For commercial licenses see https://www.tiny.cloud/
```

Comment:
I've removed a lot of old legacy code here from Firefox 9 and less, etc... here, as it just bloated the code and definitely wasn't useful.

Eg. The `load` event was added in Firefox 9 as per https://bugzilla.mozilla.org/show_bug.cgi?id=185236